File: Laboratorio hackeramk/04 C++/01_clase_objeto.cpp
#include <iostream>
using namespace std;

class Persona {
public:
    string nombre;

    void saludar() {
        cout << "Hola, soy " << nombre << endl;
    }
};

int main() {
    Persona persona;
    persona.nombre = "Pedro";
    persona.saludar();
    return 0;
}

File: Laboratorio hackeramk/03 C Sharp/01_clase_objeto.cs
using System;

class Persona
{
    public string Nombre;

    public void Saludar()
    {
        Console.WriteLine("Hola, soy " + Nombre);
    }

    static void Main()
    {
        Persona persona = new Persona();
        persona.Nombre = "Pedro";
        persona.Saludar();
    }
}
